### PR TITLE
Update aws test GitHub Actions

### PR DESCRIPTION
**Pull request by ggabernet**
Many thanks to contributing to nf-core/atacseq!

Please fill in the appropriate checklist below (delete whatever is not relevant). These are the most common things requested on pull requests (PRs).

## PR checklist
 - [ ] This comment contains a description of changes (with reason)
 - [ ] If you've fixed a bug or added code that should be tested, add tests!
 - [ ] If necessary, also make a PR on the [nf-core/atacseq branch on the nf-core/test-datasets repo]( https://github.com/nf-core/test-datasets/pull/new/nf-core/atacseq)
 - [ ] Ensure the test suite passes (`nextflow run . -profile test,docker`).
 - [ ] Make sure your code lints (`nf-core lint .`).
 - [ ] Documentation in `docs` is updated
 - [ ] `CHANGELOG.md` is updated
 - [ ] `README.md` is updated

**Learn more about contributing:** https://github.com/nf-core/atacseq/tree/master/.github/CONTRIBUTING.md

### CODE REVIEW

File: .github/workflows/awstest.yml [hunk 1]
```diff
@@ -1,37 +1,38 @@
...
+          auto-update-conda: true
+          python-version: 3.7
+      - name: Install awscli
+        run: conda install -c conda-forge awscli
+      - name: Start AWS batch job
+        env:
+          AWS_ACCESS_KEY_ID: ${{secrets.AWS_ACCESS_KEY_ID}}
+          AWS_SECRET_ACCESS_KEY: ${{secrets.AWS_SECRET_ACCESS_KEY}}
+          TOWER_ACCESS_TOKEN: ${{secrets.AWS_TOWER_TOKEN}}
+          #AWS_JOB_DEFINITION: ${{secrets.AWS_JOB_DEFINITION}}
```
Comment (drpatelh) [inline .github/workflows/awstest.yml:29]: Why is this commented out @ggabernet?

File: .github/workflows/awsfulltest.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+          TOWER_ACCESS_TOKEN: ${{secrets.AWS_TOWER_TOKEN}}
+          #AWS_JOB_DEFINITION: ${{secrets.AWS_JOB_DEFINITION}}
+          AWS_JOB_QUEUE: ${{secrets.AWS_JOB_QUEUE}}
+          AWS_S3_BUCKET: ${{secrets.AWS_S3_BUCKET}}
+        run: | # Submits job to AWS batch using a 'nextflow-4GiB' job definition. Setting JVM options to "-XX:+UseG1GC" for more efficient garbage collection when staging remote files.
+          aws batch submit-job \
+          --region eu-west-1 \
+          --job-name nf-core-atacseq \
+          --job-queue $AWS_JOB_QUEUE \
+          --job-definition nextflow-4GiB \
```
Comment (drpatelh) [inline .github/workflows/awsfulltest.yml:35]: We may not need this larger instance and the garbage collection (`-XX:+UseG1GC`) after we have implemented Paolo's config trick.

Comment (ggabernet) [reply .github/workflows/awsfulltest.yml:35]: Hi yes, that is right, but this is not yet implemented in the `awsbatch.config`. That's the same reason why the env variable $AWS_JOB_DEFINITION is commented out above.

Comment (drpatelh) [reply .github/workflows/awsfulltest.yml:35]: Ah ok. Thanks 👍

**Pull request merged**